docker-compose.yml

```diff
@@ -57,6 +57,8 @@ services:
       FLOW_DBPATH: /flowdb
     networks:
       - private
+    ports:
+      - "3569:3569"
```

Comment:
exposing this port allows us to just rely on the emulator that's part of this docker compose when testing the system with electables-smart-contracts API instead of needing to spin up and emulator separately.